tests/test_it.py

```diff
@@ -0,0 +1,2 @@
+def test_it():
+    assert True
```

Comment:
Lol, maybe at least test that the module imports or something?

Otherwise, I guess it would be nice to add a comment saying that this is a placeholder test until real tests are added later.

Reply:
Ignore, since it's fixed in #22

Reply:
See #22 which removes this file and adds real tests.